src/components/footer/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import logo from "./assets/ac_logo_24.png";
+import BottomNavigation from '@material-ui/core/BottomNavigation';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+import styles from "./footer.module.scss";
+
+
+export default function Footer() {
+
+
+  return (
+    <BottomNavigation className={styles.Footer} showLabels>
+      <BottomNavigationAction href="/home" alt="Above Curve Homepage" icon={<img src={logo} alt="Above Curve" data-testid="ac-logo"></img>} />
```

Comment:
this href doesn't work, probably better to link it to "/".  As a side note, this should probably be written into a test to ensure that it is linking to the correct place.  Here's an SO post that discusses it: https://stackoverflow.com/questions/57827126/how-to-test-anchors-href-with-react-testing-library